### DISCUSSION TITLE
Way too many navigation bars in browser

### DISCUSSION THREAD

**Discussion by du82** [status-im/status-legacy#16631] [General]
I think we need to seriously rethink how navigation is done. There's a few different navigation bar styles, some at the top, some at the bottom, and some stacked on top of each other.

Three navigation bars on the browser make is quite unusable, since the WebView window is so small, even on my large phone.

Having navigation more like a traditional web browser would be awesome.

![Screenshot_20220112-133402_Status](https://user-images.githubusercontent.com/65834069/149609394-64e3984f-0fe4-40c7-b779-825d0e2b442d.png)

**Comment by 3esmit** [comment]
Agreed

Em sáb., 15 de jan. de 2022 01:53, Du Cheng ***@***.***>
escreveu:

**Comment by jo-mut** [comment]
yeah i second that too

**Comment by guylouis** [comment]
I think this is something that was raised in the past so I'm pinging @hesterbruikman

**Comment by John-44** [comment]
Yes this issue has been an annoyance for quite some time, good news is that the design team will be looking at solutions to fix this issue as part of the Mobile App design refresh work that's just started.

**Comment by nahuhh** [comment]
> Yes this issue has been an annoyance for quite some time, good news is that the design team will be looking at solutions to fix this issue as part of the Mobile App design refresh work that's just started.  Hope to be able to share, review and discuss this work in roughly 2.5 weeks :-) 

Personally, I don't have a problem with it, but it would make more sense to condense the address bar and nav panel. I'd prefer at the bottom.


A bigger issue with the browser, is that it doesn't recognize the device back button. Pressing the back button, instead of naVigating back in the browser, it closes status.

**Comment by du82** [comment]
> Yes this issue has been an annoyance for quite some time, good news is that the design team will be looking at solutions to fix this issue as part of the Mobile App design refresh work that's just started.

I really like the UI of the app, I just wish navigation was consistent. I like the Baidu app navigation where all navigation is at the bottom of the screen on every page. It makes things easy to reach and consistent.

**Comment by du82** [comment]
any updates on this?
